helix-term/Cargo.toml

```diff
@@ -50,6 +50,8 @@ pulldown-cmark = { version = "0.8", default-features = false }
 # file type detection
 content_inspector = "0.2.4"
 
+encoding_rs = "0.8"
```

Comment:
Doesn't helix-view/helix-core re-export encoding already? We shouldn't have to import this twice

Reply:
`helix-view` used it but didn't re-export it. I re-exported it from `helix-core` and import it from there in `helix-view` and `helix-term`.